bryan-bilek.txt

```diff
@@ -0,0 +1,49 @@
+Bryan Bilek
+
+Part Two's Codepen Link: https://codepen.io/Cubguy35/pen/WNbKpeV 
+
+1. What is Semantic HTML? Giving meaning to or defining your content.
+    2. What is HTML used for? To create pages to display on the internet.
+    3. What is an attribute and where do we put it? They give extra meaning to your tags so you can manipulate what the tag does.
+    4. What is the h1 tag used for? How many times should I use it on a page? The h1 tag is used to define the most important element on the page. Ideally, use it once as it defines the most important thing.
+    5. Name two tags that have required attributes. input tag and select tag.
+    6. What do we put in the head of our HTML document? A title, styles, meta information and more.
+    7. What is an id? It specifies a unique id for an html element.
+    8. What elements can I add an id to? On any element.
+    9. How many times can I use the same id on a page? Just once.
+    10. What is a class? Can be used to specify html elements.
+    11. What elements can I add a class to? On any element.
+    12. How many times can I use the same class on a page? Multiple times. 
+    13. How do I get my link to open in a new tab? Add anchor tabs.
+    14. What is the alt attribute used for? To provide alternative info on images or for users.
+    15. How do I reference an id? Using the # symbol can reference it.
+    16. What is the difference between a section and a div? A div is a less specified tag for styling or making a section whereas section tags are more specific to the content in a body.
+    17. What is CSS used for? It's used to style webpages.
+    18. How to we select an element? Example - every h2 on the page. You could call for the h2 tag in your code or define it with a class and call that class.
+    19. What is the difference between a class and an id? - Give me an example of when I might use each one. A class can be called by a . symbol whereas an id is called by a # symbol.
+    20. How do we select classes in CSS? Write a . followed by the name of the class you want.
+    21. How do we select a p element with a single class of �human��? Call the class by writing .human
+    22. What is a parent child selector? When would this be useful? It's when you have an element or elements nested under a different class. Useful to call one more specifically.
+    23. How do you select all links within a div with the class of sidebar? Call the class .sidebar
```

Comment:
`.sidebar` would select all elements with a class of 'sidebar'.  What would be a more specific selector that includes the links as well?